cwa/olin/src/lib.rs

```diff
@@ -229,6 +276,7 @@ impl Resource {
         err::Error::check(res).map(Resource)
     }
 
+    /// wrapper for access by raw characteristics
```

Comment:
```rust
/// For the few times you _actually_ do know the file descriptor.
```

I say this because as implemented, resource ID's are randomly assigned integer values. The spec doesn't say any characteristics about them.